src/c++/uds/kernelLinux/uds/uds-sysfs.c

```diff
@@ -241,94 +222,15 @@ static const struct kobj_type memory_object_type = {
 };
 
 #endif /* TEST_INTERNAL or VDO_INTERNAL */
```

Comment:
Are the changes in uds-syfs.c and sysfs.c due to the fact that the log_level functions just go away?

It might be less confusing to split those out into a separate commit since they're internal--only changes.

Reply:
Yes... I'll split those out